Snail/Observable.swift

```diff
@@ -297,33 +297,40 @@ public class Observable<T>: ObservableType {
     public static func combineLatest<U>(_ input1: Observable<T>, _ input2: Observable<U>) -> Observable<(T, U)> {
         let combined = Observable<(T, U)>()
 
-        var value1: T?
-        var value2: U?
+        var input1Result: (value: T?, complete: Bool) = (nil, false)
```

Comment:
can this follow boolean naming of `isComplete` ? 
consider adding a typealias for the tuple to ease the maintenance cost